reindexer/reindex_worker/src/test/scala/uk/ac/wellcome/platform/reindex/reindex_worker/dynamo/BatchItemGetterTest.scala

```diff
@@ -0,0 +1,63 @@
+package uk.ac.wellcome.platform.reindex.reindex_worker.dynamo
+
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.{FunSpec, Matchers}
+import uk.ac.wellcome.json.JsonUtil._
+import uk.ac.wellcome.platform.reindex.reindex_worker.fixtures.ReindexDynamoFixtures
+
+class BatchItemGetterTest
+    extends FunSpec
+    with Matchers
+    with ScalaFutures
+    with ReindexDynamoFixtures {
+
+  it("finds a single specified record") {
+    withLocalDynamoDbTable { table =>
+      val batchItemGetter = createBatchItemGetter
+
+      val records = createRecords(table, count = 5)
+      val specifiedRecord = records.head
+
+      val futureResult =
+        batchItemGetter.get(List(specifiedRecord.id))(table.name)
+
+      whenReady(convertScalaFuture(futureResult)) { result =>
+        convertToAnyShouldWrapper(
+          result.map { fromJson[NamedRecord](_).get }.head shouldBe specifiedRecord)
+      }
+    }
```

Comment:
I'd have written this as:

```scala
whenReady(futureResult) { result =>
  result.map { fromJson[NamedRecord](_).get }.head shouldBe specifiedRecord
}
```

Is there something I'm missing with the wrappers?

Reply:
lol, this is from me button mashing in IntelliJ to show/hide implicits - looks like I pressed something that actually put them in code explicitly!